### PR TITLE
Issue/12 add unit tests for ember utils

### PR DESCRIPTION
**Pull request by michalsnik**
According to #12 

This PR adds tests for ember utilities functions.

### CODE REVIEW

File: test/utils/ember-test.js [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+    property = getProperty(`test = {
+      test() {}
+    }`);
+    assert.ok(emberUtils.isFunctionExpression(property.value));
+
+    property = getProperty(`test = {
+      test: function() {}
+    }`);
+    assert.ok(emberUtils.isFunctionExpression(property.value));
+  });
```
Comment (bardzusny) [inline test/utils/ember-test.js]: Do we want to add arrow function case here?

File: .nvmrc [hunk 2]
```diff
@@ -0,0 +1 @@
+6.3
```
Comment (bardzusny) [inline .nvmrc]: Did you consider adding `engines` to `package.json` too, just for the sake of completeness?

File: test/utils/ember-test.js (hunk 1 above)
Comment (bardzusny) [inline test/utils/ember-test.js]: I would argue that it's still valid (even if rarely used and down to personal preference) and should be detected by the linter - as simply, arrow functions are still function expressions.

Comment (michalsnik) [inline test/utils/ember-test.js]: Okay, let it be :)

**Pull request merged**